src/components/new-appointment/NewAppointment.js

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from 'react';
+import './new_Appointment.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { reserveAction } from '../../redux/my_appointment_redux/my_appointment';
+import { getDoctorsAction } from '../../redux/doctorReduce/doctors';
+
+const NewAppointment = () => {
```

Comment:
- [ ] The new styling is beautiful, but I can't add appointments.  The submit button is not working for me no matter how many times I click:

![image](https://user-images.githubusercontent.com/57421823/193158827-f3d51acf-f792-49c5-82dd-e31aee855884.png)

Kindly correct this bug to ensure your app works as intended 💪